distribution/lib/Standard/Visualization/0.0.0-dev/src/Helpers.enso

```diff
@@ -3,6 +3,9 @@ from Standard.Base import all
 from Standard.Table import Table, Column
 import Standard.Table.Data.Storage.Storage
 
+import project.Id
+import project.Id.Id as Id_Type
```

Comment:
```suggestion
import project.Id as Id_Module
import project.Id.Id
```

My thought is - the type is something that can be seen in the docs and so seeing an alias like `Id_Type` may be a little bit more confusing (ideally we should have the doc viewer resolve the alias into the proper fully qualified type, but we probably won't have that soon).

Then, the module is not really visible anywhere apart in our internal code - so any renames do not affect anything.

So in such clashing cases I suggest to rename the module and keep the type as is - as the latter is more 'public'.